hub-config/tasks.json

```diff
@@ -8,17 +8,165 @@
+                                -23,
+                                -22,
+                                -21,
+                                -20,
+                                -19,
+                                -18,
+                                -17,
+                                -16,
+                                -15,
+                                -16,
+                                -15,
+                                -14,
+                                -13,
+                                -12,
+                                -11,
+                                -10,
+                                -9,
+                                -8,
+                                -7,
+                                -6,
+                                -5,
+                                -4,
+                                -3,
+                                -2,
+                                -1,
+                                0,
+                                1,
+                                2,
+                                3,
+                                4,
+                                5,
+                                6,
+                                7 
+                            ]
                         }
                     },
-                    "output_type": {},
-                    "target_metadata":[]
+                    "output_type": {
+                        "mean": {
+                            "output_type_id": {
+                                "required": ["NA"],
+                                "optional": null
+                            },
+                            "value": {
+                                "type": "double",
+                                "minimum": 0,
+                                "maximum": 1
+                            }
+                        },
+                        "sample": {
+                            "output_type_id_params": {
+                                "is_required": false,
+                                "type": "character",
+                                "max_length": 25,
+                                "min_samples_per_task": 100,
+                                "max_samples_per_task": 100,
+                                "compound_taskid_set": ["origin_date", "location", "variant", "horizon"]
+                            },
+                            "value": {
+                                "type": "double",
+                                "minimum": 0,
+                                "maximum": 1
+                            }
+                        }
+                    },
+                    "target_metadata":[
+                        {
+                            "target_id": "variant prop",
+                            "target_name": "frequency of observed variant lineage",
+                            "target_units": "percentage",
+                            "target_keys": {
```

Comment:
@nickreich I'm having a tough time parsing this explanation of `target_keys`:

> Should be either null, in the case where the target is not specified as a task_id and is specified solely through the target_id target_metadata property or an object with one or more properties, the names of which match task_id variable(s) named within the same model_tasks object. Each property should have one specified value. Each value, or the combination of values if multiple keys are specified, define a single target value.

Do we need to specify target_keys here, if we're expecting a `variant prop` value that reflects the `compound_taskid_set`?

Reply:
I am not feeling 100% confident in this, but I think we could leave this to null and think about the target as implied by the contents of the row and target id.

Reply:
thanks--just pushed a commit with the next iteration of target_metadata